src/components/reusables/heros/HeroSingle.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { Link } from "gatsby"
```

Comment:
@yohdev-travis Can we remove all of the imports that are not being used?

Reply:
I moved the prefooter to it's on file since the home page was the only one that was using it.

Reply:
Empty imports have been removed.